libut/tests/Makefile

```diff
@@ -4,7 +4,8 @@ OBJS=$(patsubst %,%.o,$(PROGS))
 
 CFLAGS += -I../include
 CFLAGS += -g
-CFLAGS += -Wall -Wextra
+CFLAGS += -Wall -Wextra -Werror -fPIC
+CFLAGS += $(OPTFLAGS)
```

Comment:
Adding `$(OPTFLAGS)` seems harmless, I think (if no version of Make really complains about undefined variables). I'm still leery of messing with `-fPIC`, though.
Also, I just checked, and the Debian package doesn't install `libut.a` at all. I don't think the corresponding SUSE package should try to install `libut.{a,so}` either, unless some user is actually demanding it (see again: extraordinary evidence).